applications/FluidDynamicsApplication/custom_elements/embedded_fluid_element.cpp

```diff
@@ -0,0 +1,494 @@
+#include "custom_elements/embedded_fluid_element.h"
+#include "custom_elements/qs_vms.h"
```

Comment:
Why we need to include the VMS element?

Reply:
I'm commenting it out for now because it is not yet used, but once I implement the boundary integral in the QSVMS element we should have an embedded verision of VMS too, which is declared in this cpp.